services/chainstorage/src/main/java/tech/pegasys/teku/services/chainstorage/StorageService.java

```diff
@@ -149,12 +153,20 @@ protected SafeFuture<?> doStart() {
                               pruningActiveLabelledGauge));
                 }
               }
+
+              final DataArchive dataArchive =
+                  config
+                      .getBlobsArchivePath()
+                      .map(path -> (DataArchive) new FileSystemArchive(Path.of(path)))
```

Comment:
you can avoid cast via
`.<DataArchive>map(path -> new FileSystemArchive(Path.of(path)))`